tensorflow_serving/tools/docker/Dockerfile.mkl

```diff
@@ -0,0 +1,98 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG TF_SERVING_VERSION=latest
+ARG TF_SERVING_BUILD_IMAGE=tensorflow/serving:${TF_SERVING_VERSION}-devel-mkl
+
+FROM ${TF_SERVING_BUILD_IMAGE} as build_image
+FROM ubuntu:16.04
```

Comment:
Does this work with ubuntu:18.04?
We just updated our CPU images to this.

Reply:
Ah, how did i miss that. Yeah let me change that piece and test.

Reply:
We are having issues with `18.04`. So sticking to `16.04`